internal/utils/utils.go

```diff
@@ -337,24 +337,3 @@ func ExtractResourceId(id string) string {
 		return id
 	}
 }
-
-// DefaultTagsHandler converts default_tags attribute to map[string]string
-// supports only string values
-func DefaultTagsHandler(m map[string]interface{}, d *diag.Diagnostics) map[string]string {
-	newMap := make(map[string]string, len(m))
-	for key, value := range m {
-		strValue := ""
-
-		switch v := value.(type) {
-		case string:
-			strValue = v
-		default:
-			d.AddError("Client error", fmt.Sprintf("Invalid type '%v' for default_tags %s value, only string supported", v, key))
-			continue
-		}
-
-		newMap[key] = strValue
-	}
-
-	return newMap
-}
```

Comment:
https://github.com/infobloxopen/terraform-provider-bloxone/pull/124#discussion_r1626632625 -> removing the unused function